### PR TITLE
Fix: Symbols in Message Keys + Sidebar Visible on First Chat

### PR DESCRIPTION
**Pull request by cappuccinocosmico**

### CODE REVIEW

**Comment by cappuccinocosmico** [comment]
solves #68 and #65

File: frontend/src/lib/components/ChatUI.tsx [hunk 1]
```diff
@@ -185,43 +185,43 @@ function ContextSources() {
...
-    content: "...",
-    key: `${Math.floor(Math.random() * 100)}`,
-  },
-  {
-    role: "assistant",
-    content: "Pueblo colorado has .........",
-    key: `${Math.floor(Math.random() * 100)}`,
-  },
-];
+// const startingMessages: Message[] = [
```
Comment (mirrir0) [inline frontend/src/lib/components/ChatUI.tsx:191]: these can be removed

File: frontend/src/lib/components/ChatUI.tsx [hunk 2]
```diff
@@ -232,7 +232,7 @@ function MessageComponent({
   message = {
     role: "user",
     content: "",
-    key: `${Math.floor(Math.random() * 100)}`,
+    key: Symbol(),
```
Comment (mirrir0) [inline frontend/src/lib/components/ChatUI.tsx:235]: noice

File: frontend/src/lib/components/ChatUI.tsx [hunk 3]
```diff
@@ -377,7 +377,7 @@ function ChatBox() {
         h="100vh"
       >
         {messages.map((m: Message) => {
-          return <MessageComponent key={m.key} message={m} />;
+          return <MessageComponent message={m} />;
```
Comment (mirrir0) [inline frontend/src/lib/components/ChatUI.tsx:380]: did this work? afaik maps still need the key prop to be set

File: frontend/src/lib/components/DefaultShell.tsx [hunk 4]
```diff
@@ -115,24 +115,24 @@ export default function Page({ children }: { children: React.ReactNode }) {
           <SidebarSection flex="1" overflowY="auto" overflowX="hidden">
             <NavGroup>
               <NavItem href="/" icon={<FiHome />} isActive={pathIs("")}>
-                Home
+                Home (Currently Basic Chat)
```
Comment (mirrir0) [inline frontend/src/lib/components/DefaultShell.tsx:118]: add a TODO to have a better ui here

File: frontend/src/lib/components/ChatUI.tsx (hunk 1 above)
Comment (mirrir0) [inline frontend/src/lib/components/ChatUI.tsx:191]: this can be removed

**Review by mirrir0 — CHANGES_REQUESTED** [review]

File: frontend/src/lib/components/ChatUI.tsx (hunk 3 above)
Comment (cappuccinocosmico) [reply frontend/src/lib/components/ChatUI.tsx:380]: Yeah Tried it repeatedly under a bunch of different conditions and didnt find anything amiss.

File: frontend/src/lib/components/DefaultShell.tsx (hunk 4 above)
Comment (cappuccinocosmico) [reply frontend/src/lib/components/DefaultShell.tsx:118]: Rewrote and fixed in upcoming PR

**Pull request merged**